airflow/templates.py

```diff
@@ -30,3 +35,32 @@ def is_safe_attribute(self, obj, attr, value):
         ``_``) whilst still blocking internal or truely private attributes (``__`` prefixed ones).
         """
         return not jinja2.sandbox.is_internal_attribute(obj, attr)
+
+
+def ds_filter(value):
+    return value.strftime('%Y-%m-%d')
+
+
+def ds_nodash_filter(value):
+    return value.strftime('%Y%m%d')
+
+
+def ts_filter(value):
+    return value.isoformat()
+
+
+def ts_nodash_filter(value):
+    return value.strftime('%Y%m%dT%H%M%S')
+
+
+def ts_nodash_with_tz_filter(value):
+    return value.isoformat().replace('-', '').replace(':', '')
+
+
+FILTERS = {
+    'ds': ds_filter,
+    'ds_no_dash': ds_nodash_filter,
+    'ts': ts_filter,
+    'ts_no_dash': ts_nodash_filter,
+    'ts_no_dash_with_tz': ts_nodash_with_tz_filter,
```

Comment:
Bit nitpicky, but I think keeping the "no_dash" naming equal avoids any confusion.

```suggestion
def ds_no_dash_filter(value):
    return value.strftime('%Y%m%d')


def ts_filter(value):
    return value.isoformat()


def ts_no_dash_filter(value):
    return value.strftime('%Y%m%dT%H%M%S')


def ts_no_dash_with_tz_filter(value):
    return value.isoformat().replace('-', '').replace(':', '')


FILTERS = {
    'ds': ds_filter,
    'ds_no_dash': ds_no_dash_filter,
    'ts': ts_filter,
    'ts_no_dash': ts_no_dash_filter,
    'ts_no_dash_with_tz': ts_no_dash_with_tz_filter,
```

Reply:
Oh yeah, that was not an intentional change.